bin/aws-efs-creator.ts

```diff
@@ -1,21 +1,50 @@
 #!/usr/bin/env node
 import 'source-map-support/register';
+
 import * as cdk from 'aws-cdk-lib';
+import * as dotenv from 'dotenv';
+import { Aspects } from 'aws-cdk-lib';
+import { ApplyTags } from '../utils/apply-tag';
+import { checkEnvVariables } from '../utils/check-environment-variable';
+import { AwsEfsCreatorStackProps } from '../lib/AwsEfsCreatorStackProps';
 import { AwsEfsCreatorStack } from '../lib/aws-efs-creator-stack';
 
+dotenv.config(); // Load environment variables from .env file
 const app = new cdk.App();
-new AwsEfsCreatorStack(app, 'AwsEfsCreatorStack', {
-  /* If you don't specify 'env', this stack will be environment-agnostic.
-   * Account/Region-dependent features and context lookups will not work,
-   * but a single synthesized template can be deployed anywhere. */
+const appAspects = Aspects.of(app);
+
+const { CDK_DEFAULT_ACCOUNT: account, CDK_DEFAULT_REGION: region } = process.env;
+
+const cdkRegion = process.env.CDK_DEPLOY_REGION!;
+const deployEnvironment = process.env.ENVIRONMENT!;
```

Comment:
**Suggestion:** The `process.env` variables are being accessed directly with non-null assertion (`!`). This can lead to runtime errors if the environment variables are not set. Instead, provide a default value or handle the case where the variable is undefined.
> 🌟  **High Priority** 
 > [category: possible issue, importance: 8]
```suggestion
const cdkRegion = process.env.CDK_DEPLOY_REGION || 'default-region';
const deployEnvironment = process.env.ENVIRONMENT || 'development';
```